lnwallet/channel.go

```diff
@@ -163,10 +165,21 @@ type PaymentDescriptor struct {
 	// possible upstream peers in the route.
 	isForwarded bool
 
-	// pkScript is the raw public key  script that encodes the redemption
-	// rules for this particular HTLC. This field will only be populated
-	// iff the EntryType of this PaymentDescriptor is Add.
-	pkScript []byte
+	// [our|their|theirPrev]PkScript are the raw public key scripts that
+	// encodes the redemption rules for this particular HTLC. These fields
+	// will only be populated iff the EntryType of this PaymentDescriptor
+	// is Add. ourPkScript is the ourPkScript from the context of our local
+	// commitment chain. [their|theirPrev]PkScript are the two latest
+	// pkScripts from the context of the remote commitment chain.
+	ourPkScript       []byte
+	theirPkScript     []byte
+	theirPrevPkScript []byte
+
+	// isDust[Local|Remote] denotes if this HTLC is below the dust limit in
+	// locally or remotely. These fields are checked to see if an output for
```

Comment:
Minor nit: as of the latest diff of this PR, the last sentence is no longer correct.